### PR TITLE
remove inline js from layout/base template

### PR DESCRIPTION
**Pull request by achen2401**
- refactor duplicate code shared between layout.html and eproms/base.html
- remove inline js code from footer html
- remove reference to i18next in sessionMonitor.js

### CODE REVIEW

File: portal/static/js/main.js [hunk 1]
```diff
@@ -5428,13 +5431,95 @@ var tnthAjax = {
...
-    setTimeout('$("#homeFooter").show();', 100);
+    setTimeout(function() { $("#homeFooter").show(); }, 100);
+
+    setTimeout(function() {
+        var userLocale = $("#copyrightLocaleCode").val();
+        var footerElements = "footer .copyright, #homeFooter .copyright, .footer-container .copyright";
+        var getContent = function(cc) {
+            var content = "";
+            switch(String(cc.toUpperCase())) {
+                case "US":
```
Comment (ivan-c) [inline portal/static/js/main.js]: Awesome! Wasn't sure the best way to tackle this, but this is a great start.

File: portal/static/js/sessionMonitor.js [hunk 2]
```diff
@@ -23,14 +23,14 @@ var SessionMonitorObj = function(crsf_token, session_lifetime) {
...
+    
     window.sessMon = sessMon;
-    var warningText = (i18next.t("Your session will expire in approximately {time} seconds due to inactivity.")).replace("{time}",(sessMon.timeBeforeWarning / 1000));
     $("#session-warning-modal").modal({"backdrop": false,"keyboard": false,"show": false})
             .on("show.bs.modal", function() { sessMon.modalShown = true; })
             .on("hide.bs.modal", function() { sessMon.modalShown = false; if (sessMon.intervalMonitor)  { clearInterval(sessMon.intervalMonitor); } })
             .on("click", "#stay-logged-in", sessMon.extendsess)
             .on("click", "#log-out", sessMon.logout)
-            .find("#remaining-time").text(warningText);
+            .find("#remaining-time").text("Your session will expire in approximately {time} seconds due to inactivity.".replace("{time}",(sessMon.timeBeforeWarning / 1000)));
```
Comment (ivan-c) [inline portal/static/js/sessionMonitor.js]: Does this need to be wrapped in `t()`?

**Comment by achen2401** [comment]
@ivan-c ivan, thank for taking a look at my changes. I think I addressed the issue you raised.  I encountered a js error when i was testing so I also fixed that.  let me know if you found any more issue with my changes.

**Comment by achen2401** [comment]
Ah finding more errors - will test before re-open this again.

**Pull request closed without merging**